Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'mongo_mapper', '0.12.0'
+gem 'mongo', '1.11.1'
```

Comment:
It's the version we use so I wanted to ensure that in development of this gem were developing against the same version